wheel/src/adapt_response.rs

```diff
@@ -1,39 +1,26 @@
+use std::rc::Rc;
+
 use crate::lazy_node::LazyNode;
 use clvmr::allocator::Allocator;
-use clvmr::reduction::{EvalErr, Response};
-
-use std::rc::Rc;
+use clvmr::reduction::Response;
 
 use pyo3::prelude::*;
-use pyo3::types::PyDict;
 
-pub fn eval_err_to_pyresult<T>(py: Python, eval_err: EvalErr, allocator: Allocator) -> PyResult<T> {
-    let node = LazyNode::new(Rc::new(allocator), eval_err.0);
-    let msg = eval_err.1;
-    let ctx: &PyDict = PyDict::new(py);
-    ctx.set_item("msg", msg)?;
-    ctx.set_item("node", node)?;
-    Err(py
-        .run(
-            "
-from clvm.EvalError import EvalError
-raise EvalError(msg, node)",
-            None,
-            Some(ctx),
-        )
-        .unwrap_err())
-}
-
-pub fn adapt_response_to_py(
+pub fn adapt_response(
     py: Python,
     allocator: Allocator,
-    r: Response,
-) -> PyResult<(u64, LazyNode)> {
-    match r {
+    response: Response,
+) -> PyResult<(PyObject, LazyNode)> {
+    match response {
         Ok(reduction) => {
             let val = LazyNode::new(Rc::new(allocator), reduction.1);
-            Ok((reduction.0, val))
+            let rv: PyObject = reduction.0.into_py(py);
+            Ok((rv, val))
+        }
+        Err(eval_err) => {
+            let rv: PyObject = eval_err.1.into_py(py);
+            let val = LazyNode::new(Rc::new(allocator), eval_err.0);
+            Ok((rv, val))
```

Comment:
so, this is not returning an `Err()`, does that mean that the `deserialize_and_run_program()` won't throw an exception on errors, but return an error?
That seems like a significant change in the API, especially since there are still failures that would cause it to fail, just not `EvalErr`.

Reply:
That's true. I can change the name if necessary. The thing is I believe no one is using the `clvm_rs` wheel any longer, so API changes seem pretty safe. At least, `chia-blockchain` is not using it.

Reply:
I've removed old api and created a brand new name for use going forward: `run_serialized_chia_program`.